tests/chocolatey-tests/commands/choco-search.Tests.ps1

```diff
@@ -35,7 +36,8 @@ Describe "choco <_>" -ForEach $Command -Tag Chocolatey, SearchCommand, FindComma
         Remove-ChocolateyTestInstall
     }
 
-    Context "Searching packages with no filter (Happy Path)" {
+    # Skip on Licensed Extension when searching against a v3 source as our source is not returning consistent results.
+    Context "Searching packages with no filter (Happy Path)" -Skip:$hasV3Feed {

```

Comment:
If I'm understanding correctly, this means _all_ of these tests will be skipped completely if any one configured source has a v3 feed url?

Can we disable the v3 feed url and use other urls? Or is that not feasible in our current configuration?

Reply:
In the Test Kitchen environment as it exists today, we don't really have more than one test feed enabled during the discovery phase, and we also currently only test against either the v2 or the v3 feed, but not against both in the same run.

Just to be a little more explicit about what we're skipping, I've updated the variable name to be `$hasEnabledV3Feed` since that's what we're checking against. Theoretically the setup feed that we pull Chocolatey and other such packages from could be v3 while the test is against v2.